packages/salesforcedx-vscode-core/src/commands/forceRefreshSObjects.ts

```diff
@@ -145,7 +145,7 @@ export class ForceRefreshSObjectsExecutor extends SfdxCommandletExecutor<{}> {
     const commandName = execution.command.logName;
     try {
       let result;
-      if (response.data.source === SObjectRefreshSource.StartupMin) {
+      if (response.data.source === SObjectRefreshSource.Startup) {
```

Comment:
We have two different sobject refresh startup flows;`SObjectRefreshSource.Startup` which uses the `enable-sobject-refresh-on-startup` setting and the expectation is to download all custom objects and sobjects, the other flow is `SObjectRefreshSource.StartupMin` which is triggered by the user clicking on a pop-up message on the bottom right corner when there are no sobjects present in the project and it only creates 16 sobjects.

I don't think we should do this since that would force the flow that uses `enable-sobject-refresh-on-startup` setting to just create sobjects from cache and the pop-up message flow to sync all the objects from the org.

Reply:
Agree! We want both these to only download a list of min. For some reason when I earlier searched for StartupMin usage, I didn't find the pop-up message code path. I see it now. I'll take care of this.

Reply:
One question - If we are going to make the enable-sobject-refresh-on-startup to be the default behavior as part of ther next story, do we even need the pop-up route to generate sobjects?

Reply:
I have made the change to make both the code paths work the same. As part of the next story, I can get rid of the pop-up route to generate sobjects.